package-lock.json

```diff
@@ -3364,7 +3364,7 @@
         },
         "string_decoder": {
           "version": "1.1.1",
-          "resolved": "https://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
+          "resolved": "http://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
```

Comment:
Why they went from `https` to `http` 😕  ?

Reply:
It's a painful known bug from npm: https://npm.community/t/package-lock-json-keeps-changing-between-platforms-and-runs/1129/8

But there's a suggestion so I might try it.

Reply:
Just tried it and it even replaces existing `http:`. Yay!